### PR TITLE
add logo to Readme and doc

### PR DESCRIPTION
**Pull request by m-abdollahi**
ADD logo to README and DOC index.md

# Description:

## Type of change:

- [ ] Bug fix.
- [x] New feature.
- [x] Documentation update.

## Checklist:

This checklist can be used as a help for the reviewer.

- [x] Is the code easy to read and understand?
- [x] Are comments for humans to read, not computers to disregard?
- [ ] Does a new feature has an accompanying new test (in the CI or unit testing schemes)?
- [x] Has the documentation been updated as necessary?
- [x] Does this close the issue?
- [ ] Is the change limited to the issue?
- [ ] Are errors handled for all outcomes?
- [ ] Does the new feature provide new restrictions on dependencies, and if so is this documented?

## Comments:

- [x] ADD logo to README.MD file #287 
- [x] ADD logo to Doc/Index.MD file #287

### CODE REVIEW

**Review by CasperWA — CHANGES_REQUESTED** [review]
Thanks, but please make these links relative and avoid fencing it in a `p` tag - there is no need, use instead the `align` attribute of the `img` tag. Furthermore, it will probably look nicer if it's aligned right.

**Review by CasperWA — CHANGES_REQUESTED** [review]
Could you move this line up to be the first line in `README.md` as this doesn't look very good at the moment:
image

Please also run:
```shell
pip install pre-commit invoke
SKIP=pylint pre-commit run --all-files
```
And commit the changed files afterwards.

**Comment by m-abdollahi** [comment]
Okay, but I did not understand what you mean :)

File: README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # EMMOntoPy
 
+ 
+<img src="docs/images/suspected_logo.png" width="40%" align="middle">
```
Comment (CasperWA) [inline README.md:4]:
```suggestion
<img src="docs/images/suspected_logo.png" width="40%" align="right">

# EMMOntoPy

```

Comment (CasperWA) [reply README.md:4]: This is what I meant.

**Review by CasperWA — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
+<img src="docs/images/suspected_logo.png" width="40%" align="right">
+
 # EMMOntoPy
 
+
```
Comment (CasperWA) [inline README.md:5]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,11 @@
+<img src="docs/images/suspected_logo.png" width="40%" align="right">
+
 # EMMOntoPy
 
+ 
 *Python API for the Elemental Multiperspective Material Ontology ([EMMO]).*
 
+
```
Comment (CasperWA) [inline README.md:8]:
```suggestion
```

File: docs/index.md [hunk 4]
```diff
@@ -1,7 +1,11 @@
+<img src="images/suspected_logo.png" width="40%" align="right">
+
 # EMMOntoPy
 
+
```
Comment (CasperWA) [inline docs/index.md:5]:
```suggestion
```

File: docs/index.md [hunk 5]
```diff
@@ -1,7 +1,11 @@
...
 *Python API for the Elemental Multiperspective Material Ontology ([EMMO]).*
 
+
```
Comment (CasperWA) [inline docs/index.md:8]:
```suggestion
```

**Review by CasperWA — CHANGES_REQUESTED** [review]
No reason to add extra lines. After this this looks great :)

**Review by CasperWA — APPROVED** [review]
This is great - thank you @m-abdollahi ! :)

**Pull request merged**